docs/python/debugging.md

```diff
@@ -99,11 +101,18 @@ You would then use the following configuration to attach from the VS Code Python
     "name": "Python: Attach",
     "type": "python",
     "request": "attach",
-    "host": "localhost",
-    "port": 5678,
+    "connect": {
+        "host": "localhost",
+        "port": 5678
+    },
+    "listen": {
```

Comment:
If you start debugpy with `--listen`, the debug config should only contain `"connect"`. And vice versa, `--connect` requires `"listen"`. Using both side by side wouldn't be meaningful, and the debugger will report an error if this is attempted.

Reply:
Ok, corrected in latest commit.